pkg/org/auth.go

```diff
@@ -116,7 +116,7 @@ func (m *Middleware) UserAndProject(next bunrouter.HandlerFunc) bunrouter.Handle
 func (m *Middleware) userFromRequest(req bunrouter.Request) *bunconf.User {
 	ctx := req.Context()
 
-	if len(m.userProviders) == 0 {
+	if len(m.userProviders) == 0 && m.app.Config().UserProviders.OIDC == nil {
```

Comment:
This seems to be unneeded?

Reply:
It's because `OIDCProvider` doesn't implement `UserProvider`. The OIDC/SSO flow works by accessing `/v1/api/sso/oidc/callback` with the right parameters, which then gets exchanged and produces an Uptrace `token` cookie (same as with user/pass login).

Originally I tried to make this fit with the `UserProvider` middleware, but the SSO flow happens separately. Let me know if you have suggestions to unify with the `JWTProvider`/`CloudflareProvider`?

Reply:
Yep, good catch